PyDynamic/uncertainty/propagate_filter.py

```diff
@@ -51,86 +58,98 @@ def FIRuncFilter(y,sigma_noise,theta,Utheta=None,shift=0,blow=None):
     """
     if not isinstance(sigma_noise, float):
         raise NotImplementedError(
-            "FIR formula for covariance propagation not implemented yet. Suggesting Monte Carlo propagation instead.")
-    Ncomp = len(theta) - 1      # FIR filter order
+            "FIR formula for covariance propagation not implemented yet. "
+            "Suggesting Monte Carlo propagation instead.")
+    Ncomp = len(theta) - 1  # FIR filter order
 
-    if not isinstance(Utheta, np.ndarray):      # handle case of zero uncertainty filter
+    if not isinstance(Utheta,
```

Comment:
Just to be sure, the comment does not effect the line break in the string for the statement in lines 60-62.

Reply:
I would not call it "code improvement". But instead of 

``` python
if not isinstance(Utheta,
                  np.ndarray):  # handle case of zero uncertainty filter
```
i would suggest:
``` python
# handle case of zero uncertainty filter
if not isinstance(Utheta, np.ndarray):
```

Reply:
Agree, that's what I did.